guides/common/modules/proc_recovering-corrupted-content-on-a-smart-proxy.adoc

```diff
@@ -0,0 +1,34 @@
+[id="Recovering_Corrupted_Content_on-a-Smart-Proxy_{context}"]
+= Recovering corrupted content on a {SmartProxy}
+
+In case of content corruption on a {SmartProxy}, you can recover it by using the `verify-checksum` command in Hammer CLI.
+The `verify-checksum` command works for optional parameters associated with the {SmartProxy} such as `content-view id`, `environment-id`, or `repository-id` to limit the repair.
+You can run the `verify-checksum` command without any filters so it runs on all content on the {SmartProxy}.
+
+.CLI procedure
+* To run a `verify-checksum` command in Hammer CLI for `content-view id`:
```

Comment:
Again, instead of describing the command, let's say what it does:
```suggestion
* To repair content in a content view, run Hammer on your {SmartProxy}:
```
Similarly for the rest.